library_generation/README.md

```diff
@@ -0,0 +1,112 @@
+# Generate GAPIC Client Library without post-processing
+
+The script, `generate_library.sh`, allows you to generate a GAPIC client library from proto files.
+
+## Environment
+
+Use Linux environment and install java runtime environment (8 or above).
+
+## Prerequisite
+In order to generate a GAPIC library, you need to pull `google/` from [googleapis](https://github.com/googleapis/googleapis)
+and put it into the directory containing `generate_library.sh` since protos in `google/` 
+are likely referenced by protos from which the library are generated.
+
+In most cases, the `proto_path` is a subdirectory within `google/`.
```

Comment:
Is it a requirement? Does the script fail if `proto_path` is not a subdirectory within `google/`?

Reply:
No. Showcase client generation is an example that `proto_path` is outside of `google/`.

Reply:
I think we should remove this sentence from `Prerequisite` section if it is not required.

Reply:
done